src/settings/target/manifest.rs

```diff
@@ -92,30 +103,25 @@ impl Manifest {
             None => Some(true),
         };
 
+        template_config.name = name;
```

Comment:
I'm trying to understand how this logic maps onto the removed logic that creates a new Manifest--mind filling me in?

Reply:
this will read in the manifest from the template and then override individual fields with what was passed via the command line. so for this - name is taken from `wrangler generate` and overrides whatever name was previously serialized

Reply:
since we're serializing a manifest in the first place i decided to just make it mutable rather than making a new one and cloning everything, should be more performant

Reply:
Cool!